docs/sphinx/source/api.rst

```diff
@@ -311,6 +311,13 @@ PVWatts model
    inverter.pvwatts
    pvsystem.pvwatts_losses
 
+Loss models
```

Comment:
Let's move this to the "Effects on PV System Output" section, OK to keep it as a subheading there.

If you agree, the outline hierarchy would be;

Effects on PV System Output
    Loss models
        pvsystem.combine_loss_factors
    snow.coverage_nrel

etc.